packages/esm-commons-lib/src/components/encounter-tile/encounter-tile.component.tsx

```diff
@@ -2,41 +2,23 @@ import { openmrsFetch } from '@openmrs/esm-framework';
 import { CodeSnippetSkeleton, Tile, Column } from '@carbon/react';
 import React, { useEffect, useState } from 'react';
 import styles from './encounter-tile.scss';
-import {
-  artStopDateUUID,
-  artTherapyDateTime_UUID,
-  dateRestartedUUID,
-  encounterRepresentation,
-  substitutionDateUUID,
-  switchDateUUID,
-} from '../../constants';
-import moment from 'moment';
-import { findObs, getObsFromEncounter } from '../../index';
+import { encounterRepresentation } from '../../constants';
 
 export interface EncounterTileColumn {
   key: string;
   header: string;
   encounterUuid: string;
-  concept: string;
-  isConceptDate?: Boolean;
-  isConceptSummaryDate?: Boolean;
-  isSummaryDaysCalculation?: Boolean;
-  obsValue?: string;
-  summaryConcept?: string;
-  summaryObsValue?: string;
-  isARTDateConcept?: Boolean;
+  getObsValue: (encounter: any) => string;
+  getSummaryObsValue?: (encounter: any) => string;
+  encounter?: any;
```

Comment:
do we need both `encounterUuid` and `encounterUuid`? what's the difference

Reply:
I didn't get this. You wrote the same thing